android/library/src/main/java/io/logto/android/authflow/browser/BrowserAuthFlow.kt

```diff
@@ -0,0 +1,114 @@
+package io.logto.android.authflow.browser
```

Comment:
`authflow` isn't a valid word. from the naming convention i can see, does `io.logto.android.auth.BrowserFlow` and `io.logto.android.auth.NativeFlow` make more sense?

Reply:
good